radix-engine-tests/tests/application/metering.rs

```diff
@@ -417,11 +418,13 @@ fn run_basic_transfer_to_virtual_account(mode: Mode) {
 }
 
 fn run_radiswap(mode: Mode, use_v1_1_pools: bool) {
```

Comment:
Minor, but maybe worth changing the name of this variable? Not sure if `use_v1_1_pools` is clear.
eg.
`use_v1_1_pools` -> `pool_math_precision_fix`

Reply:
Yep